### PR TITLE
refac: Use core registrations and remove duplications

### PR DESCRIPTION
**Pull request by RasmusGraabaek**

### CODE REVIEW

File: source/B2CWebApi/Program.cs [hunk 1]
```diff
@@ -12,48 +12,58 @@
...
 using Energinet.DataHub.EDI.ArchivedMessages.Application.Extensions.DependencyInjection;
 using Energinet.DataHub.EDI.B2CWebApi.Extensions.DependencyInjection;
 using Energinet.DataHub.EDI.B2CWebApi.Security;
 using Energinet.DataHub.EDI.IncomingMessages.Application.Extensions.DependencyInjection;
-using Microsoft.ApplicationInsights.Extensibility;
 
 var builder = WebApplication.CreateBuilder(args);
 
-const string domainName = "EDI.B2CWebApi";
+const string domainName = "EDI";
```
Comment (RasmusGraabaek) [inline source/B2CWebApi/Program.cs:30]: This is currently the same as in our API project, is that a problem?

**Review by ebbeknudsen — COMMENTED** [review]
Isn't SQL server health checks missing in all our modules?

File: source/BuildingBlocks.Application/Extensions/DependencyInjection/FileStorageExtensions.cs [hunk 2]
```diff
@@ -44,6 +47,15 @@ public static IServiceCollection AddFileStorage(this IServiceCollection services
 
         services.AddTransient<IFileStorageClient, DataLakeFileStorageClient>();
 
+        var uri = configuration["AZURE_STORAGE_ACCOUNT_URL"];
```
Comment (ebbeknudsen) [inline source/BuildingBlocks.Application/Extensions/DependencyInjection/FileStorageExtensions.cs:50]: Description / comment

File: source/IntegrationTests/Infrastructure/Authentication/MarketActors/JwtBuilder.cs [hunk 3]
```diff
@@ -24,7 +25,7 @@ namespace Energinet.DataHub.EDI.IntegrationTests.Infrastructure.Authentication.M
     public class JwtBuilder
     {
         private static readonly JwtSecurityTokenHandler _tokenHandler = new();
-        private static readonly byte[] _defaultSigningKey = Guid.NewGuid().ToByteArray();
+        private static readonly byte[] _defaultSigningKey = Guid.NewGuid().ToByteArray().Concat(Guid.NewGuid().ToByteArray()).ToArray();
```
Comment (ebbeknudsen) [inline source/IntegrationTests/Infrastructure/Authentication/MarketActors/JwtBuilder.cs:28]: Is this a mistake? Or what does this change do?

**Review by ebbeknudsen — DISMISSED** [review]

**Review by ebbeknudsen — APPROVED** [review]

**Pull request merged**